pkg/internal/firebase/firebase.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 Google LLC
```

Comment:
let's put these in just

/internal

I don't want anyone depending on our repo to provide this.

Reply:
They are 😄